database/migrations/2024_04_19_131906_add_index_to_poly_line_parent_id.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('poly_lines', function (Blueprint $table) {
+            $table->index('parent_id');
+        });
+    }
+
+    public function down(): void
+    {
+        Schema::table('poly_lines', function (Blueprint $table) {
+            $table->dropIndex('parent_id');
```

Comment:
Did you checked, that reverting is working? 
As I know giving a string here would look for a index with the given name. I'd use `['parent_id']` as argument here.